lib/feed.js

```diff
@@ -0,0 +1,45 @@
+const data = require('../out/data.json')
+const feedFinder = require('find-rss')
+const fs = require('fs')
+const feedParser = require('feed-read-parser')
+
+const errs = []
+var articles = []
+
+function getFeeds(feed_urls, i, cb) {
+  var feed_url = feed_urls[i]
+  if (!feed_url) return cb(errs, articles)
+  feedParser.get(feed_url, function(err, _articles) {
+    if (err) errs.push(err.message + ' ' + feed_url)
+    else {
+      articles = articles.concat(_articles)
+    }
+    getFeeds(feed_urls, i + 1, cb)
+  })
+}
+
+async function getFeedLinks(data) {
+  const linkPromises = data.map(org => {
+    const { blog_url } = org
+    if (blog_url) {
+      return feedFinder(blog_url)
+        .catch(err => console.error(err))
+        .then(res => {
+          if (res.length > 0) {
+            return res[0].url
+          }
+        })
+    }
+  })
+  return await Promise.all(linkPromises)
+}
+
+;(async () => {
+  const feedLinks = (await getFeedLinks(data)).filter(link => link != undefined)
```

Comment:
if you're checking for `null` **and** `undefined` **and** empty string, just `link => link` is enough.